addons/website_livechat/security/ir.model.access.csv

```diff
@@ -2,3 +2,5 @@ id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_im_livechat_channel_public,im_livechat.channel.public,im_livechat.model_im_livechat_channel,,1,0,0,0
 access_website_visitor_livechat_users,website.visitor.livechat.users,model_website_visitor,im_livechat.im_livechat_group_user,1,1,0,0
 access_website_track_livechat_users,website.track.livechat.users,website.model_website_track,im_livechat.im_livechat_group_user,1,0,0,0
+access_website_page_livechat_users,website.page.livechat.users,website.model_website_page,im_livechat.im_livechat_group_user,1,0,0,0
+access_website_view_livechat_users,website.view.livechat.users,base.model_ir_ui_view,im_livechat.im_livechat_group_user,1,0,0,0
```

Comment:
That seems a bit strange.
Why not applying the corresponding necessary groups on the fields / form views then to avoid errors?

I'm not super confortable allowing people to read technical models such as ir.ui.view.
EDIT: Well I see on the PR that @mart-e agrees.